tests/serial/watch-integral/WatchIntegralScaleAndNoScale.cpp

```diff
@@ -98,6 +98,7 @@ BOOST_AUTO_TEST_CASE(WatchIntegralScaleAndNoScale)
       std::string fileName = "precice-SolverTwo-watchintegral-WatchIntegral.log";
       auto        result   = readDoublesFromTXTFile(fileName, 3);
       auto        expected = std::vector<double>{
+          0.0, 6.5, 3.0,
```

Comment:
I would have expected `0.0, 0.0, 0.0` here. Not sure where the `6.5` comes from. @fsimonis any idea?

Reply:
Checked this again: I think the last column is clear to me now. We consider two edges with a total length of `3.0`. The second column still looks dubious to me.

Reply:
If I change the coupling scheme in the test to `parallel-implicit` the results look like I would expect them. This also means that the output of the integrals depends on the coupling scheme. I will open a bug report in a moment since this kind of behavior looks wrong to me.

Reply:
See #2187 for more information.